web/src/reducers/dataReducer.js

```diff
@@ -1,13 +1,21 @@
 const { isEmpty } = require('lodash');
 const moment = require('moment');
 
-const { modeOrder } = require('../helpers/constants');
 const constructTopos = require('../helpers/topos');
 const translation = require('../helpers/translation');
 
 const exchangesConfig = require('../../../config/exchanges.json');
 const zonesConfig = require('../../../config/zones.json');
 
+function computeZoneAggregates(zone) {
+  zone.maxProduction = Math.max(...Object.values(zone.production || {}));
+  zone.maxDischarge = -Math.min(Math.min(...Object.values(zone.storage || {})), 0);
+  zone.maxStorage = Math.max(Math.max(...Object.values(zone.storage || {})), 0);
+  zone.maxExport = -Math.min(Math.min(...Object.values(zone.exchange || {})), 0);
+  zone.maxImport = Math.max(Math.max(...Object.values(zone.exchange || {})), 0);
+  return zone;
+}
```

Comment:
This logic looks quite different now (in particular the console warnings have been removed), what's the reasoning there? :slightly_smiling_face: